### ISSUE TITLE
How to make subsequent API calls using Oraclize from smart contracts?

### ISSUE THREAD

**Issue by mandritac** [oraclize/ethereum-bridge#47]
Hi,

Oraclize first query is free as per documentation, then how can we make more subsequent calls?

Thanks.

**Comment by D-Nice** [comment]
As per https://docs.oraclize.it/#pricing-advanced-datasources-call-fee 

The fee is taken at time of `oraclize_query` method call, from the calling contract. Therefore, all you need to do is ensure that the contract has a sufficient ether balance. We have some examples in our example repo showing a check that ensures there's enough ether before trying to do the query.

**Comment by mandritac** [comment]
Yeah I have also put a check to ensure the query is not called if there is no enough ether balance. However how much ether should I mention in the oraclize_query so that I can make subsequent calls? In the document - https://docs.oraclize.it/#ethereum-quick-start-custom-gas-limit-and-gas-price I can see that custom gas price is set programmatically in the contract. However can you please guide how much wei should I mention?

Thanks.

**Comment by D-Nice** [comment]
Here's an example on how to get the relevant ETH needed to make the next call: https://github.com/oraclize/ethereum-examples/blob/master/solidity/KrakenPriceTicker.sol#L32

Both our docs and examples cover this.
